### DISCUSSION TITLE
If we calibrate to Intervention Admissions and allow Entering Samples to stratify by Intervention, are those counted as new Admissions?

### DISCUSSION THREAD

**Discussion by MJC598** [SyndemicsLab/respond#20] [Q&A] [answered]
@ddbaptiste As the title states, if we allow Entering Cohorts to be stratified by Intervention should those be considered "New Admissions" in regards to hitting targets?

**Comment by ddbaptiste** [comment, marked as answer]
I think that they **should** count as new admissions to treatment, as that's what I understand the target data to represent.

This will definitely make calibration (and input generation/manipulation in general) more difficult, since there will be more values, but I do think it's the correct decision.
